_posts/2019-11-13-AWS-Dev-Day-2019.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+authors: [yolan_vloeberghs]
+title: 'AWS Dev Day 2019'
+image: /img/aws-dev-day-2019-banner.png
+tags: [AWS, Cloud, Conference]
+category: Conference
+comments: true
+---
+# AWSome Dev Day 2019!
+
+>The AWS Dev Day is a free conference day hosted by Amazon where architects and developers from Amazon and other companies talk about the most popular AWS services, the newest features and everything related to cloud development. These sessions explain a lot about what AWS has to offer and you will see live demos on how you can implement them in your own use cases. 
+
+This year, AWS decided to host their annual Dev Day conference in Belgium, more specifically Mechelen. This was very convenient us, as our headquarters are located in Mechelen aswell. This means that there was a very good reason for Ordina consultants to attend the event. We went with a small group of 8 people to make sure we didn't miss the latest AWS news. We were treated with delicious food & drinks throughout the day to make sure their guests were happy.
```

Comment:
AWS hosts devdays across the world every year. This was just the BE event.
So a more correct wording would be:
Amazon held their Belgian devdays in Mechelen this year. This was very convenient for us as the Ordina Belgium HQ is also located in Mechelen.